lighthouse/classes/event_properties/definitions.py

```diff
@@ -257,3 +280,224 @@ def value(self):
 
     def add_to_warehouse_message(self, message):
         message.add_metadata("source_plate_barcode", self.value)
+
+
+class CherrytrackWellsFromDestination(EventPropertyAbstract, ServiceCherrytrackMixin):
```

Comment:
I think namespacing instead of adding 'Cherrytrack' at the beginning would be more sustainable.

Reply:
I find this getting confusing if they are all properties of the event? Maybe some distinction is needed between the properties are extracted from the parameters coming from the request and the properties that are then subsequently retrieved from cherrytrack?

Reply:
Yes, not good to have Cherrytrack in the name, I am thinking on just removing it and then having WellsFromDestination.

Reply:
Did you move this?

Reply:
Yes, now is moved under 'biosero' subfolder